website_sale_mandate/views/sale_order_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="inherit_view_order_form_inherit_sale_mandate" model="ir.ui.view">
+        <field name="name">website_sale_mandate.view_order_form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" positon="inside">
+                <page name="SEPA" string="SEPA" attrs="{'invisible': [('authorize_iban_direct_debit','=', False)]}">
+                    <group>
+                        <field name="authorize_iban_direct_debit" invisible="1"/>
+                        <field name="show_mandate_action" invisible="1"/>
+                        <field name="iban"/>
+                        <field name="iban_contact"/>
+                    </group>
+                </page>
+            </xpath>
+            <button name="preview_sale_order" position="before">
+                <button name="action_create_partner_mandate" type="object" string="Create SEPA Mandate" class="btn-primary" attrs="{'invisible': [('show_mandate_action', '=', False)]}"/>
```

Comment:
Improvement idea: have a "Pending validation" information on the sale order, in this case : 
- user has clicked on button "Create SEPA Mandate"
- the SEPA mandate is not validated yet

Because currently when we click on the button, there is no change on the screen, we do not know if the process has worked, and the button is still displayed (then it means we can launch the process several times in parallel).

The proposition here is that, if Odoo find a mandate for this partner with the same iban but not validated, it hides the "Create SEPA mandate" button and display a "Pending mandate validation" tag somewhere.